package.json

```diff
@@ -16,11 +16,20 @@
     "start": "react-scripts start",
     "test": "react-scripts test"
   },
+  "resolutions": {
+    "//node-gyp": "5.0.0"
```

Comment:
Your Netlify preview build failed : https://app.netlify.com/sites/carbon-tutorial/deploys/5d406b79c73f55000c9a9d98 

Seems to be related to this.

Reply:
11:08:51 AM: warning Resolution field "//node-gyp" does not end with a valid package name and will be ignored.
This is not the issue , I comit the resolution the build process is not restarting

Reply:
OK - 
Why are you including these extra packages? Recommend only adding the packages included in the tutorial step: https://www.carbondesignsystem.com/tutorial/react-step-1

Reply:
I was having issues with the yarn add node-sass, which is why I re ran the yarn install by removing lock files and played around. Is there a way can I close this PR and rework on this . If there is work around to rework on this initial step I would like to finish this asap. In My local its working and building fine . In Git this is failing this check .
Thanks,

Reply:
Yes - that's fine

Reply:
I am not able to do anything on this branch or not letting me push any changes to this step I already completed 2 3 4 5 steps . Can you please suggest me.

Reply:
I would do the following: 

1.  Close this PR.
2. `git checkout master`
3. `git branch -D react-step-1`
4. `git fetch upstream`
5. `git checkout -b react-step-1 upstream/react-step-1`

Then follow the the rest of the instructions in step 1.